product_code_builder/models/product_code.py

```diff
@@ -0,0 +1,63 @@
+# -*- coding: utf-8 -*-
+###############################################################################
+#
+#   Module for OpenERP
+#   Copyright (C) 2015 Akretion (http://www.akretion.com). All Rights Reserved
+#   @author Benoît GUILLOT <benoit.guillot@akretion.com>
+#
+#   This program is free software: you can redistribute it and/or modify
+#   it under the terms of the GNU Affero General Public License as
+#   published by the Free Software Foundation, either version 3 of the
+#   License, or (at your option) any later version.
+#
+#   This program is distributed in the hope that it will be useful,
+#   but WITHOUT ANY WARRANTY; without even the implied warranty of
+#   MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#   GNU Affero General Public License for more details.
+#
+#   You should have received a copy of the GNU Affero General Public License
+#   along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+###############################################################################
+
+from openerp import fields, models
+
+
+def update_null_and_slash_codes(cr):
```

Comment:
Just rebase and it will pass.  Squash a few commits while you are at it ;) 

Sent from TypeMail



On 2 Jul 2015 9:49 PM, at 9:49 PM, David Beal <notifications@github.com> wrote:
>> +#   License, or (at your option) any later version.
>> +#
>> +#   This program is distributed in the hope that it will be useful,
>> +#   but WITHOUT ANY WARRANTY; without even the implied warranty of
>> +#   MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
>> +#   GNU Affero General Public License for more details.
>> +#
>> +#   You should have received a copy of the GNU Affero General Public
>License
>> +#   along with this program.  If not, see
><http://www.gnu.org/licenses/>.
>> +#
>>
>+###############################################################################
>> +
>> +from openerp import fields, models
>> +
>> +
>> +def update_null_and_slash_codes(cr):
>
>No problem, it's my fault. I asked @ahilali to copy your code here to
>pass runbot. But it was a bad idea.
>He'll remove it. Thanks
>
>---
>Reply to this email directly or view it on GitHub:
>https://github.com/OCA/product-attribute/pull/70/files#r33762439